README.md

```diff
@@ -161,64 +163,80 @@ local body, metadata = lcmark.convert("Hello *world*",
                          "latex", {smart = true, columns = 40})
 ```
 
-The module exports
+The module exports the following fields:
 
 - `lcmark.version`: a string with the version number.
 
 - `lcmark.yaml_parser_name`: a string holding the name of the
-    automatically-loaded module and function used to parse YAML. Possible values
-    are:
-    - `lyaml.load` (lyaml)
-    - `yaml.load` (yaml)
-    - `yaml.eval` (lua-yaml)
-    - `nil` (none) (the value `nil`, not a string)
+  automatically-loaded module and function used to parse YAML. 
+  Possible values are:
+  - `lyaml.load` (lyaml)
+  - `yaml.load` (yaml)
+  - `yaml.eval` (lua-yaml)
+  - `nil` (none) (the value `nil`, not a string)
```

Comment:
Would it be clear to say
```
- `"lyaml.load"` (lyaml)
- etc.
- `nil`
```
That way the strings are marked as strings, and we don't need the special comment for nil?

Reply:
Yup, good point.

Reply:
Anything else?